custom_components/opnsense/sensor.py

```
@@ -595,38 +622,44 @@
-            return False
-
-        if prop_name in ["stddev", "delay", "loss"]:
+        try:
             value = gateway[prop_name]
-            if isinstance(value, str):
-                value = re.sub(r"[^0-9\.]*", "", value)
-                if len(value) < 1:
-                    return False
+            # cleanse "ms", etc from values
+            if prop_name in ["stddev", "delay", "loss"]:
+                if isinstance(value, str):
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)